src/PavValHackathon.Web.API/Initializers/ServiceDBInitializer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Reflection;
+using GodelTech.Microservices.Core;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using PavValHackathon.Web.Data.Contexts;
+
+namespace PavValHackathon.Web.API.Initializers
+{
+    public class ServiceDBInitializer : MicroserviceInitializerBase
+    {
+        public ServiceDBInitializer(IConfiguration configuration) : base(configuration)
+        {
+        }
+
+        public override void ConfigureServices(IServiceCollection services)
+        {
+            var migrationsAssembly = typeof(Startup).GetTypeInfo().Assembly.GetName().Name;
+            var connectionString = Configuration.GetConnectionString("write");
+
+            services.AddDbContext<DataContext>(o => o.UseSqlServer(connectionString,
```

Comment:
dafaq is => ????????

Reply:
lambda xD
`() => {...};` == `[] () {...};`